src/components/seeds/database/EditColumns.tsx

```diff
@@ -217,10 +210,8 @@ const sections: Section[] = [
         COLUMNS_INDEXED.viabilityTests_seedsSown,
         COLUMNS_INDEXED.viabilityTests_viabilityTestResults_seedsGerminated,
         COLUMNS_INDEXED.cutTestSeedsEmpty,
-        COLUMNS_INDEXED.latestViabilityPercent,
       ],
       [
-        COLUMNS_INDEXED.latestViabilityTestDate,
         COLUMNS_INDEXED.viabilityTests_substrate,
         COLUMNS_INDEXED.cutTestSeedsCompromised,
         COLUMNS_INDEXED.viabilityTests_notes,
```

Comment:
I think we could move viabilityTests_percentGerminated to this group (this is just visual, so we don't have an empty space)